--- a/far/src/cfg/ConfigOpt.cpp
+++ b/far/src/cfg/ConfigOpt.cpp
@@ -177,7 +177,7 @@ static struct FARConfig
 	{1, NSecInterface, "TimeSeparator",             &Opt.strTimeSeparator, GetTimeSeparatorDefaultStr()},
 	{1, NSecInterface, "DecimalSeparator",          &Opt.strDecimalSeparator, GetDecimalSeparatorDefaultStr()},
 
-	{1, NSecInterface, "OSC52ClipSet",              &Opt.OSC52ClipSet, 0, REG_BOOLEAN},
+	{1, NSecInterface, "OSC52ClipSet",              &Opt.OSC52ClipSet, 1, REG_BOOLEAN},
 	{1, NSecInterface, "TTYPaletteOverride",        &Opt.TTYPaletteOverride, 1, REG_BOOLEAN},
 
 	{0, NSecInterface, "ShowTimeoutDelFiles",       &Opt.ShowTimeoutDelFiles, 50},
